components/renku_data_services/session/api.spec.yaml

```diff
@@ -253,6 +253,106 @@ paths:
           $ref: "#/components/responses/Error"
       tags:
         - session_launchers
+  /builds:
+    post:
+      summary: Create a new container image build
```

Comment:
suggestion: Since this only works on an environment. And we already have the listing of build for an environment. I think it makes sense to move this post on the same path as `/environments/{environment_id}/builds`

Reply:
:+1: I will consider this. Note that it makes the POST body empty.

Reply:
> Note that it makes the POST body empty.

Yes that is fine.

Reply:
Done. Should we validate that the body is empty? It is accepting any body payload at the moment.